install_debian_autopsy_4.19.2.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+sudo apt install wget
+
+#install dependencies
+wget https://github.com/sleuthkit/sleuthkit/releases/download/sleuthkit-4.11.1/sleuthkit-java_4.11.1-1_amd64.deb
+wget https://github.com/labcif/autopsy-packager/releases/download/4.19.2-1/autopsy_4.19.2-1_amd64.deb
```

Comment:
These calls will produce leftovers. I recommend creating a temporary folder with `mktemp -d` and trap it to safely free the resources.

Reply:
I wonder if the users want to keep the .debs for later reinstall ?

Reply:
I would argue that it is not the use case here, or at least rather a specific use case. If we want to support it, we can use something like bash `getopt` and use an option or a combination of options. We can pass options by piping the script to bash via VFS (`bash <(curl https://github.com/labcif/autopsy-packager/releases/download/4.19.2-1/script.sh) -- --keep --user`).

That, although, will inevitably write the files owned as root, hence `--user` flag. That can be mitigated using a `sudo` call in a subshell. `--user` will allow running the script as user mode but enforce it to be an interactive shell in order to `sudo` be able to ask for the password correctly.

`--keep` would imply or enforce the use of `--user`.

Reply:
Ok, use the temp dir.